tef-impl/src/main/java/flipkart/tef/bizlogics/DataAdapterBizlogic.java

```diff
@@ -80,6 +81,11 @@ private Map<DataAdapterKey<?>, Field> buildCacheOfMutableFields() {
 
     @SuppressWarnings("rawtypes")
     public static String getEmittedDataName(Class<? extends DataAdapterBizlogic> clazz) {
```

Comment:
Can you check once if this method really has to be public? If not, can we make it private and run the test via reflection, or make it package private atleast.

Reply:
Its been used in FlowBuilder also.